plugins/modules/efs.py

```diff
@@ -99,6 +99,12 @@
             - How long the module should wait (in seconds) for desired state before returning. Zero means wait as long as necessary.
         default: 0
         type: int
+    transition_to_ia:
+        description:
+            - How many days before objects transition to the lower-cost EFS Infrequent Access (IA) storage class. If set to None, any existing lifecyle policy will be removed, and objects will not transition to an IA storage class. If this parameter is absent or left empty, any existing lifecycle policy will not be affected.
```

Comment:
```suggestion
            - How many days before objects transition to the lower-cost EFS Infrequent Access (IA) storage class.
            - If set to the string C(None), any existing lifecyle policy will be removed, and objects will not transition
              to an IA storage class.
            - If this parameter is absent, any existing lifecycle policy will not be affected.
```

Rather than one long line, the string can be broken into multiple array elements, or split using standard YAML multi-line strings https://yaml-multiline.info/